pom.xml

```diff
@@ -343,6 +347,34 @@
                 <version>${version.log4j}</version>
             </dependency>
 
+            <!-- Testing utilities -->
+            <dependency>
+                <groupId>org.testcontainers</groupId>
+                <artifactId>testcontainers</artifactId>
+                <version>${version.testcontainers}</version>
```

Comment:
consider importing the BOM:
https://www.testcontainers.org/#managing-versions-for-multiple-testcontainers-dependencies

Reply:
Done.